src/iframe/src/Iframe.js

```diff
@@ -575,6 +585,7 @@ class Iframe extends Component {
 
         // Draw this state.
         window.draw && window.draw(this.store.getState())
+        this.drawPixels()
```

Comment:
I wonder if there's a way we can create a wrapper draw function so we avoid calling `drawPixels` after every `window.draw`?